astropy/coordinates/tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+import warnings
+
+
+# autouse makes this an all-coordinates-tests fixture
+@pytest.fixture(autouse=True)
```

Comment:
I guess this should be undone when #7928 is addressed (in the long run) and we can then turn most (if not all) warnings to errors?

Reply:
Yes, good point.  will add a reference to #7928 in here so we have at least a chance of remembering.